client/app/telecom/pack/slots/emailPro/detail/pack-emailPro-detail.controller.js

```diff
@@ -0,0 +1,98 @@
+angular.module('managerApp').controller('PackEmailProDetailCtrl', class PackEmailProDetailCtrl {
+  constructor(
+    $http, $q, $stateParams, $state, $translate,
+    OvhApiPackXdsl, OvhApiXdsl, PACK_EMAILPRO, Toast, URLS,
```

Comment:
The params are passed in an order different than it is done in `pack.controller.js`. We should probably keep the same order everywhere

Reply:
Alphabetical Order ($ first)